sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastExchangeExec.scala

```diff
@@ -265,7 +265,10 @@ abstract class GpuBroadcastExchangeExecBase(
   @transient
   private val timeout: Long = SQLConf.get.broadcastTimeout
 
-  val _runId: UUID = UUID.randomUUID()
+  // Cancelling a SQL statement from Spark ThriftServer needs to cancel
```

Comment:
how does it relate to #1760?

Reply:
yes :)

Reply:
yeah generally speaking would be better to separate issues in separate prs.

Reply:
note its fine for now but something to keep in mind for future

Reply:
Thanks @tgravescs  and @gerashegalov. Will post separate PR's for separate issues in the future.